Resources/public/js/components/price-list/main.js

```diff
@@ -122,10 +130,23 @@ define([], function() {
             var bulkPriceComponents = [];
 
             this.sandbox.util.foreach(this.options.currencies, function(currency){
+
+                this.bulkPriceData = [];
+
+                this.bulkPriceData.currencyCode = currency.code;
+
+                if (!!this.groupedPrices.prices) {
+                    this.bulkPriceData.prices = this.groupedPrices.prices[currency.code];
+                }
+             
+                if (!!this.groupedPrices.specialPrices && currency.code in this.groupedPrices.specialPrices) {
+                    this.bulkPriceData.specialPrice = this.groupedPrices.specialPrices[currency.code].pop();
```

Comment:
load data (specialPrice and prices) for component initialization for each currency...